main.go

```diff
@@ -167,12 +192,20 @@ func (s *Appender) archiveFileName() string {
 	return s.filePath + "_" + ts
 }
 
+// Append inserts line at the end of file and asks file to be rotated if it is too big.
 func (s *Appender) Append(line string) {
 	if s.bytesWritten >= *maxFileSize {
 		s.rotateFile()
 	}
 	n, _ := s.writer.WriteString(line)
```

Comment:
i wonder why this is `line` and not `line + "\n"`. this way you can also save `s.writer.WriteByte('\n')` and only need `s.bytesWritten += n`

Reply:
Discussed 2f2.